src/po/sr.po

```diff
@@ -6337,12 +6438,186 @@ msgstr "откуцајте  :help register<Ентер>   за информаци
 msgid "menu  Help->Sponsor/Register  for information    "
 msgstr "мени  Помоћ->Спонзор/Региструј се  за информације    "
 
+msgid ""
+"\n"
+"# Buffer list:\n"
+msgstr ""
+"\n"
+"# Листа бафера:\n"
+
+#, c-format
+msgid ""
+"\n"
+"# %s History (newest to oldest):\n"
+msgstr ""
+"\n"
+"# %s Историја (од најновијег ка најстаријем):\n"
+
+msgid "Command Line"
+msgstr "Командна линија"
+
+msgid "Search String"
+msgstr "Стринг за претрагу"
+
+msgid "Expression"
+msgstr "Израз"
+
+msgid "Input Line"
+msgstr "Линија за унос"
+
+msgid "Debug Line"
+msgstr "Debug линија"
+
+msgid ""
+"\n"
+"# Bar lines, copied verbatim:\n"
+msgstr ""
+"\n"
+"# Преградне линије, копиране дословно:\n"
+
+#, c-format
+msgid "%sviminfo: %s in line: "
+msgstr "%sviminfo: %s у линији: "
```

Comment:
Again, shouldn't it be "na liniji"?

Reply:
Thanks for your comments and suggestions, Boris.
There are a lot more strings that needs improvement.
I did this quite fast, and without spell checking, so errors were inevitable.
Currently I'm working on translation of help. So when that's finished I will go back to improve messages.

Reply:
Yeah... reference documentation is big. Good luck!